.git-blame-ignore-revs

```diff
@@ -1,5 +1,7 @@
 # Migrate code style to Black
 7ac461a3191fc56909da8ac3c8294f4dd4e0d543
+# Apply isort
+11207757b81157c13070c491ffa60822dcd33a57
```

Comment:
Maybe it'll be easier to rebase if this is one commit ? So once you hit the conflict, you checkout `ours` and then run both black and isort, otherwise you have to first check the conflicting commit to pick the right tool ?

Reply:
Sure, I'll rebase those together and update the hash

Reply:
(which I guess is the case anyway, otherwise it's hard to run the tools anyway. nevermind)

Reply:
Yep, it'll be the big commit and then the hash update and that's the end